docs/web3-eth-miner.rst

```diff
@@ -0,0 +1,231 @@
+.. _eth-miner:
+
+.. include:: include_announcement.rst
+
+==============
+web3.eth.miner
+==============
+
+
+The ``web3-eth-miner`` package allows you to remote control the node's mining operation and set various mining specific settings.
+
+
+.. code-block:: javascript
+
+    import Web3 from 'web3';
+    import {Miner} from 'web3-eth-miner';
+
+    // "Web3.givenProvider" will be set if in an Ethereum supported browser.
+    const miner = new Miner(Web3.givenProvider || 'ws://some.local-or-remote.node:8546', null, options);
+
+
+    // or using the web3 umbrella package
+    const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546', null, options);
```

Comment:
The web3-eth-miner module shouldn't be a part of the ``web3.eth`` object.

Reply:
Yes, right @nivida. I will remove the umbrella package.